scripts/eject.js

```diff
@@ -17,17 +18,32 @@ const ownDependencyNames = [
     'mkdirp'
 ];
 
+function getPromptMessage() {
+    const baseMessage =
+        'Are you sure you want to eject? This flavor is designed so that you do not have to do so, if I forgot a use case, please open an issue at\n\nhttps://github.com/cyclejs-community/one-fits-all\n\nThis process is NOT reversable, it will remove all traces of the flavor from your project\n';
+
+    if (usingPnpm)
+        return (
+            baseMessage +
+            '\nYou are using pnpm which will cause this script to delete your shrinkwrap.yaml and node_modules and reinstall afterwards\n'
+        );
+    if (usingYarn)
+        return (
+            baseMessage +
+            '\nYou are using yarn which will cause this script to delete your yarn.lock and node_modules and reinstall afterwards\n'
```

Comment:
This should be another function (e.g. arrow fn const) that takes the cli and the lockfile as input

Reply:
Sure I look into this monday